staging/src/k8s.io/apiserver/plugin/pkg/audit/dynamic/dynamic.go

```diff
@@ -159,6 +160,7 @@ func NewBackend(c *Config) (audit.Backend, error) {
 type backend struct {
 	// delegateUpdateMutex holds an update lock on the delegates
 	delegateUpdateMutex  sync.Mutex
+	delegateStopped      bool
```

Comment:
The whole backend is stopped, not just delegates, no?

Reply:
Since `*backend.stopAllDelegates()` are in `*backend.Run`, called after `<-stopCh`, I think you are right